cli.py

```diff
@@ -59,3 +59,29 @@ def prompt_subject():
     ]
     answers = prompt(questions)
     return answers['subject']
+
+
+def prompt_date():
```

Comment:
The new function could take in a paramter message that asks the user to select the date.